lib/plugins/EPrints/Plugin/Screen/ExportToOrcid.pm

```diff
@@ -456,6 +671,21 @@ sub eprint_to_orcid_work
 		}
 	}
 
+    #put-code
+    my $user = $self->{processor}->{orcid_user};
+    my @orcid_putcodes = @{ $eprint->value( "orcid_putcodes" ) };
+    if( @orcid_putcodes )
+    {
+        foreach my $pc (@orcid_putcodes)
+        {
+            if( $pc->{orcid} eq $user->value("orcid") )
+            {
+                $work->{"put-code"} = $pc;
```

Comment:
This is wrong. It should read at least `$work->{"put-code"} = $pc->{putcode};`, however that gives me a `400 Bad Request: The put code provided is not valid.` response from ORCID. Any help is much appreciated.

Reply:
Fixed.